modules/howtos/examples/provisioning-resources-collections.rb

```diff
@@ -0,0 +1,100 @@
+require "couchbase"
+include Couchbase # rubocop:disable Style/MixinUsage for brevity
+
+
+def getCollections(username, password)
+    puts "create-collection-manager"
+
+    #  tag::create-collection-manager[]
+    options = Cluster::ClusterOptions.new
+    options.authenticate(username, password)
+    cluster = Cluster.connect("couchbase://localhost", options)
+    bucket = cluster.bucket("travel-sample")
+    collections = bucket.collections
+    # end::create-collection-manager[]
+
+    return collections
+end
+
+def main
+    options = Cluster::ClusterOptions.new
+    options.authenticate("Administrator", "password")
+    cluster = Cluster.connect("couchbase://localhost", options)
+    # bucket = cluster.bucket("travel-sample")
```

Comment:
Not sure if we need this since it's commented out?

Reply:
good spot ta